x-pack/plugins/apm/typings/es_schemas/raw/event_raw.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { APMBaseDoc } from './apm_base_doc';
+import { TimestampUs } from './fields/timestamp_us';
+
+export interface Processor {
+  name: 'event';
+  event: 'event';
+}
+
+export interface EventRaw extends APMBaseDoc {
+  processor: Processor;
```

Comment:
Does the EventRaw have the processor field?

Reply:
Good point, it doesn't. I'll remove it

Reply:
It's updated now!